qemu/tests/cfg/boot_from_device.cfg

```diff
@@ -1,92 +1,81 @@
 - boot_from_device:
     no Host_RHEL.m5, Host_RHEL.m6.u0, Host_RHEL.m6.u1, Host_RHEL.m6.u2
+    virt_test_type = qemu
     type = boot_from_device
     boot_menu = on
     enable_sga = yes
-    image_boot = no
-    virt_test_type = qemu
-    boot_menu_key = "f12"
-    Host_RHEL.m7:
-        boot_menu_key = "esc"
+    boot_menu_key = "esc"
+    Host_RHEL.m6:
+        boot_menu_key = "f12"
     boot_menu_hint = "Press .*(F12|ESC) for boot menu"
     boot_entry_info = "Booting from Hard Disk..."
-    boot_fail_info = "Boot failed: not a bootable disk"
+    images = "stg"
+    image_name_stg = "images/stg"
+    image_size_stg = 100M
+    force_create_image_stg = yes
+    remove_image_stg = yes
     variants:
-        - boot_from_hard_drive:
-            dev_name = hard-drive
-            bootindex_image1 = 0
-        - boot_from_usb_stg:
+        - cdrom:
+            cdroms = "test"
+            cdrom_test = "/tmp/test.iso"
```

Comment:
It is a absolute path,  but when call it in `create_cdroms` and `clean_cdroms`, `utils_misc.get_path(data_dir.get_data_dir()`  be used again,  it is redundant.
Can choose any one:
1. use relative path here, such as `test.iso`
2. remove `utils_misc.get_path`